website/docs/r/fsx_lustre_file_system.html.markdown

```diff
@@ -24,7 +24,7 @@ resource "aws_fsx_lustre_file_system" "example" {
 
 The following arguments are supported:
 
-* `storage_capacity` - (Required) The storage capacity (GiB) of the file system. Minimum of `3600`. Storage capacity is provisioned in increments of 3,600 GiB.
+* `storage_capacity` - (Required) The storage capacity (GiB) of the file system. Minimum of `1200`. Storage capacity is provisioned in increments of 1,200 GiB.
```

Comment:
Minimum increment is still 3,600 but minimum size is 1,200

Reply:
Oops, good catch! Thanks